### PR TITLE
For 31

### PR DESCRIPTION
**Pull request by Globolobo**

### CODE REVIEW

File: client/app/components/ServicesTemplate/index.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import json from './data.json';
+
+// Styles
+import styles from './styles.scss';
+
+const ServicesTemplate = () => {
+  const filledCardsTop = json.data.topCardContent.map(({ title, subtitle, content, url, img, id }) =>
+    <Card
+      key={id}
+      cardSize = "is-6"
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/index.js]: Are you sure AirBnB linter rules are working on your setup? Stuff like spaces around equals in assigning props or not having spaces between object keys should be popping up.
screen shot 2018-01-05 at 8 25 39 am

File: client/app/components/ServicesTemplate/index.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    Each service within the collection verifies one or more consumer 
+    identity attributes using the internal restricted-use-only databases of 
+    landline, VoIP and wireless telephone service providers.`;
+
+  return (
+    <section>
+      <section className={cn('hero is-fullheight is-primary', styles.aboutHero)}>
+        <div className="hero-body">
+          <div className="container">
+            <div className="cardsContainer">
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/index.js:41]: Class `cardsContainer ` doesn't exist. div is still needed to wrap the elements, but the class either was forgotten or not necessary

File: client/app/components/SharedComponents/Card/index.js [hunk 3]
```diff
@@ -34,12 +34,13 @@ const SolutionCard = ({ subtitle, title, img, content, url }) => (
   </div>
 );
 
-SolutionCard.propTypes = {
+Card.propTypes = {
   subtitle: PropTypes.string.isRequired,
```
Comment (stern-shawn) [inline client/app/components/SharedComponents/Card/index.js]: Consider removing isRequired, or add subtitles to the topCardContent elements.

screen shot 2018-01-05 at 8 37 37 am

File: client/app/components/ServicesTemplate/data.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{
+  "data":
+    {
+      "topCardContent":
+        [{
+          "id": "1",
+          "title": "Name and Address",
+          "content": "Verify telephone numbers, accounts, email addresses, names, SSN, and addresses",
+          "url": "telified",
+          "img": "3D"
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/data.json:10]: screen shot 2018-01-05 at 8 31 41 am

Images not provided, I assume we need to get fresh svgs from @leseanbiz or can we reuse existing ones for this section?

File: client/app/components/ServicesTemplate/index.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+// Styles
+import styles from './styles.scss';
+
+const ServicesTemplate = () => {
+  const filledCardsTop = json.data.topCardContent.map(({ title, subtitle, content, url, img, id }) =>
+    <Card
+      key={id}
+      cardSize = "is-6"
+      {...{ title, subtitle, content, url, img}}
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/index.js]: No `subtitle` keys defined in `data.json`

File: client/app/components/ServicesTemplate/index.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    />
+  );
+
+  const filledCardServices = json.data.serviceCardContent.map(({id, title, subtitle, url}) =>
+    <ServicesCardContent
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/index.js]: Array of elements, should use id as key to suppress React warnings

File: client/app/components/ServicesTemplate/index.js [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+    <section>
+      <section className={cn('hero is-fullheight is-primary', styles.aboutHero)}>
+        <div className="hero-body">
+          <div className="container">
+            <div className="cardsContainer">
+              <div className="container">
+                <div className="card animated fadeInUp">
+                  <div className="card-content">
+                    <div className="content has-text-centered">
+                      <h2 style = {{paddingBotom: '2em'}}>
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/index.js]: lul Botom

File: client/app/components/ServicesTemplate/index.js [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+                        Our Featured Service: <h1 style = {{marginBottom: '0'}}>The Telified Collection</h1>
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/index.js]: Valid DOM nesting, no h1s within other h*s, or putting card lists inside of p tags

screen shot 2018-01-05 at 8 47 59 am

File: client/app/components/ServicesTemplate/index.js [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+// Styles
+import styles from './styles.scss';
+
+const ServicesTemplate = () => {
+  const filledCardsTop = json.data.topCardContent.map(({ title, subtitle, content, url, img, id }) =>
+    <Card
+      key={id}
+      cardSize="is-6"
+      {...{ title, subtitle, content, url, img }}
```
Comment (stern-shawn) [inline client/app/components/ServicesTemplate/index.js]: No subtitles defined, not sure if expected or not so making subtitle optional prop

**Comment by stern-shawn** [comment]
Commit e19f50f will hopefully merge correctly with the other PR

File: client/app/components/ServicesTemplate/data.json (hunk 4 above)
Comment (leseanbiz) [reply client/app/components/ServicesTemplate/data.json:10]: i will try and get svgs ASAP for this and the other services below.

**Pull request merged**